config/prod/dgutman-htan-ec2.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v7.yaml
+stack_name: dgutman-htan-ec2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "HTAN"
+  # The resource owner
+  OwnerEmail: "xengie.doan@sagebase.org"
+  # Default account settings, leave alone unless you plan to override
+  VpcName: {{stack_group_config.default_vpc}}
+  KeyName: EmoryHTAN-dgutman ### have to override
+  # KeyName: {{stack_group_config.default_key_pair}}
+  # AMIId: {{stack_group_config.default_ami}}
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "m5d.xlarge"
+  # (Optional) ID of the base AMI (default: ami-97785bed) (supported distros: AWS linux)
+  AMIId: "ami-0de53d8956e8dcf80"      # Amazon Linux 2 AMI (HVM), SSD Volume Type
```

Comment:
It was set to this in the yaml that David Gutman sent me on gmail, but I'm not sure if he had a reason for it.

Reply:
actually, i think the default one (ami-0080e4c5bc078760e) would be better since it it has a bunch of things already installed including docker which is what he wants.

Reply:
that's fine-- I copied what was recommended on aws for the machine type I had requested,